client/src/components/App/App.jsx

```diff
@@ -230,27 +254,21 @@ function App() {
     try {
       await axios.post(`${API_BASE_URL}/users/logout`);
       localStorage.removeItem("current_user_id");
+      localStorage.removeItem("current_doctor_id");
```

Comment:
question - do we need more than user_id? As in, why do we also need a doctor id for? 

Food for thought of, can we remove doctor_id?

Reply:
Oh this is a good point I will look more into this.